app/models/__init__.py

```diff
@@ -886,6 +887,14 @@
     ]
 }
 
+
+STATISTICS_VALID_KEYS = {
+    "GET": [
+        CREATED_KEY,
+        ID_KEY,
+        VERSION_KEY
+    ]
+}
 TEST_CASE_VALID_KEYS = {
```

Comment:
Missing a blank line here, but again that looks like it was moved by mistake.

Reply:
Corrected